stubs/sublime.pyi

```diff
@@ -2471,7 +2471,7 @@ class View:
 
     def export_to_html(
         self,
-        regions=None,
+        regions: Optional[Region | List[Region]] = None,
```

Comment:
```suggestion
        regions: Region | List[Region]] | None = ...,
```

Just for consistency. I used `Optional` only at two places inside of tuples, because `Foo | None` for a single tuple element is a bit difficult to read.